base/docs/Docs.jl

```diff
@@ -201,7 +201,7 @@ end
 
 doc(f, ::Method) = doc(f)
 
-# Modules
+# Modules
```

Comment:
It's a non-breaking space. We have a lot of them.

I have an old PR to clean them up but I still need to rebase it and check if any of them are used intentionally.